OMNA/registerpage.html

```diff
@@ -288,6 +288,600 @@
             </table>
         </div>
 
+        <div>
+            <h2>OMA Security</h2>
+            <table id="registries-table" class="table-bordered">
```

Comment:
I think you are overusing "id".

id should be unique on a page. You have many "registries-table" id's.

There should also be a reason for giving an element an id. Such as for styling, or for Javascript.

Please remove any id's that are not unique, or have no use.

Thank you